Content.Shared/Mobs/Systems/MobStateSystem.Subscribers.cs

```diff
@@ -77,11 +77,11 @@ private void OnStateExitSubscribers(EntityUid target, MobStateComponent componen
                 //unused
                 break;
             case MobState.Critical:
-                _standing.Stand(target);
+                //_standing.Stand(target);    DeltaV - Remove Redundancy
```

Comment:
```suggestion
                //_standing.Stand(target); // DeltaV - Remove Redundancy
```

This is usually how these are formatted